.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
+known_numpy=numpy
+known_keras=keras
+known_concurrent=concurrent.futures
+sections=CONCURRENT,FUTURE,STDLIB,KERAS,NUMPY,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
```

Comment:
What is the `sections` parameter doing? Can you point me to the documentation where you found it?

Reply:
> It sets the order of imports
> Link : https://pypi.python.org/pypi/isort under Custom Sections and Ordering

OK, we definitely don't want this. Why should concurrent always come before anything else?

Reply:
I'll look into ways of creating import exceptions for specific files.

Reply:
Why creating exceptions for specific files?

Reply:
The same config is going to be used for the entire repo. So if we don't want to set concurrent.future as the highest priority import globally, we'll need to set it up as an exception.

Reply:
To be clear, what I'm asking is not about concurrent.futures specifically.
```
from concurrent.futures import ThreadPoolExecutor
import glob
```
and not:
```
import glob
from concurrent.futures import ThreadPoolExecutor
```

OR

```
from aSTANDARDLIBRARY import SOMETHING
import glob
```
and not:
```
import glob
from aSTANDARDLIBRARY import SOMETHING
```

We want to do this consistently without using exceptions, it should be the default for the entire project and not only for concurrent.futures but for anything else too.

Reply:
I can set the priority for standard libraries to be higher than that for glob

Reply:
`glob` is a standard library too. I don't want to make glob always appear after other standard libraries. I want `glob` to appear after `concurrent.futures` because `glob` starts with `g` and `concurrent.futures` starts with `c`. But `glob` should still appear before other standard libraries whose name starts with a letter > `g`.